src/web/components/Callout/Checkboxes.tsx

```diff
@@ -0,0 +1,65 @@
+import React, { useState } from 'react';
+
+import { CheckboxGroup, Checkbox } from '@guardian/src-checkbox';
+import { FieldLabel } from './FieldLabel';
+
+type Props = {
+    formField: CampaignFieldCheckbox;
+    formData: { [key in string]: any };
+    setFormData: React.Dispatch<React.SetStateAction<{ [x: string]: any }>>;
+};
+
+// We encapsulate the Component's state here instead of directly sourcing it from
+// `formData` to avoid unnessesary rerenders, as this was causing selection animation
+// to be applied whenever formData was changing
+export const Checkboxes = ({ formField, formData, setFormData }: Props) => {
+    const [state, setState] = useState([]);
+    const updateState = (newState: []) => {
+        setState(newState);
+        setFormData({
+            ...formData,
+            [formField.id || '']: state,
+        });
+    };
+
+    return (
+        <>
+            <FieldLabel formField={formField} />
+            {formField.options && (
+                <CheckboxGroup name={formField.name || ''}>
+                    {formField.options.map((option, index) => {
+                        const checkboxSelection =
+                            formField.id && formField.id in formData
+                                ? formData[formField.id]
+                                : [];
+                        const isCheckboxChecked = checkboxSelection.find(
+                            (ele: string) => ele === option.value,
+                        );
+                        return (
+                            <Checkbox
+                                key={index}
+                                label={option.value}
+                                value={option.value}
+                                name={`${formField.id}`}
```

Comment:
Should this be name={formField.name}?

Minus the string literal selectors and using `name`